uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/JwtBearerGrantIT.java

```diff
@@ -98,11 +102,11 @@ public void testJwtBearerGrantForUnknownClient() {
 
```

Comment:
Should we add an integration test for the existent client not authorized for the device?  Similar to testNoAllowedDeviceIdSet unit test...